pages/news/templates/news/detail.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+
+<article>
+    <h3 class="font-medium">{{ news.title }}</h3>
+    <p class="text-sm text-pp-dark-grey mb-6">{{ news.created_at|date:"F d, Y" }}</p>
+    <img src="{{ news.image.url }}" alt="{{ news.title }}" class="w-full h-64 sm:h-96 object-cover rounded-lg">
+
+    <div>
```

Comment:
Something like
```suggestion
    <div class="mt-6">
```
for better visual spacing?